packages/blade/docs/utils/useTheme.stories.mdx

```diff
@@ -0,0 +1,108 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Utils/useTheme" />
+
+## useTheme <!-- omit in TOC -->
+
+- [Overview](#overview)
+- [Usage](#usage)
+- [Types](#types)
+  - [`Theme`](#theme)
+  - [`ColorScheme`](#colorscheme)
+  - [`Platform`](#platform)
+  - [`ColorSchemeInput`](#colorschemeinput)
+
+### Overview
+
+`useTheme` is a custom React hook provided by the `@razorpay/blade/components`  that allows you to access the current theme and color scheme of your application.
```

Comment:
now that we're properly documenting this, should we expose this from `@razorpay/blade/utils` instead? that's where other hooks are exported from so this seems a bit confusing.


If we want to avoid breaking change, we can probably export from both and document the `@razorpay/blade/utils` everywhere 🤔

Reply:
Makes sense. Putting it up for a vote.

Reply:
what was the conclusion lets put that as well?

Reply:
Yes, put it up for a vote - got max votes for putting it in both places

Reply:
I've updated the documentation also accordingly